website/pages/api-docs/system/replication/index.mdx

```diff
@@ -146,14 +164,31 @@ performnace secondary and DR primary node, it will look something like:
       "last_wal": 455,
       "merkle_root": "cdcf796619240ce19dd8af30fa700f64c8006e3d",
       "mode": "primary",
-      "primary_cluster_addr": ""
+      "primary_cluster_addr": "",
+      "secondaries": [
+        {
+          "api_address": "https://127.0.0.1:49277",
+          "cluster_address": "https://127.0.0.1:49281",
+          "connection_status": "connected",
+          "last_heartbeat": "2020-06-10T15:40:46-07:00",
+          "node_id": "4"
+        }
+      ]
     },
     "performance": {
       "cluster_id": "1598d434-dfec-1f48-f019-3d22a8075bf9",
       "known_primary_cluster_addrs": ["https://127.0.0.1:8201"],
       "last_remote_wal": 291,
       "merkle_root": "43f40fc775b40cc76cd5d7e289b2e6eaf4ba138c",
       "mode": "secondary",
+      "primaries": [
+        {
+          "api_address": "https://127.0.0.1:49244",
+          "cluster_address": "https://127.0.0.1:8201",
+          "connection_status": "connected",
+          "last_heartbeat": "2020-06-10T15:40:46-07:00"
```

Comment:
I think it would be helpful to have a note somewhere about the possible values of connection status and what the last heartbeat represents (basically is it from the perspective of this node, or the other node).

Reply:
Added in https://github.com/hashicorp/vault/pull/9215/commits/2b8fe3306a2022d9fc5c19d925e903c80cebd123